env0/resource_api_key.go

```diff
@@ -59,14 +98,19 @@ func resourceApiKey() *schema.Resource {
 func resourceApiKeyCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	apiClient := meta.(client.ApiClientInterface)
 
+	organizationRole := d.Get("organization_role").(string)
+	if organizationRole == organizationRoleAdmin && len(d.Get("project_permissions").(*schema.Set).List()) > 0 {
+		return diag.Errorf("project_permissions cannot be set when organization_role is Admin")
```

Comment:
not required.. .but validates schema level that not using "Admin" with "project_permissions" (In the UI this is not a valid option).